roles/dhcp/templates/subnets.j2

```diff
@@ -0,0 +1,77 @@
+# This file is autogenerated by Ansible {{ ansible_version.full }}
```

Comment:
can we rename this file to something else than `subnets` - maybe something like `dhcpd_conf.j2`